packages/help-center/src/components/help-center-chat-history.tsx

```diff
@@ -9,10 +9,40 @@ import NavItem from 'calypso/components/section-nav/item';
 import NavTabs from 'calypso/components/section-nav/tabs';
 import { HELP_CENTER_STORE } from '../stores';
 import { HelpCenterSupportChatMessage } from './help-center-support-chat-message';
+import { getFilteredConversations } from './utils';
 import type { ZendeskConversation } from '@automattic/odie-client';
 
 import './help-center-chat-history.scss';
 
+const Conversations = ( { conversations }: { conversations: ZendeskConversation[] } ) => {
+	const { __ } = useI18n();
+	if ( ! conversations || ! conversations.length ) {
+		return <div className="help-center-chat-history__no-results">{ __( 'Nothing found…' ) }</div>;
+	}
```

Comment:
I'd possibly do not show anything if we don't have old chats.
Wdyt @lucasmendes-design?

<img width="442" alt="image" src="https://github.com/user-attachments/assets/73e289b4-aa27-47ca-9281-e2b346c91fb0">